module2/homework/addEven/addEven.hpp

```diff
@@ -2,7 +2,10 @@
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+	int sum {};
+	for (auto e : numbers) {
```

Comment:
I suggest you using `auto&` or even `const auto&`
```suggestion
	for (cons auto& e : numbers) {
```